test/runtests.jl

```diff
@@ -1,4 +1,4 @@
-using Test, Sandbox
+using Test, Sandbox, UserNSSandbox_jll
```

Comment:
It's annoying how a project needs to depend on a package for Preferences to pick up the preference. Is that intended?

Reply:
Yeah, I (mistakenly) thought you needed the import too.